forte/data/ontology/ontology_code_generator.py

```diff
@@ -601,6 +579,28 @@ def parse_schema(self, schema: Dict,
                 self.allowed_types_tree[en.class_name].add(
                     property_name)
 
+    def parse_onto_ref(self, user_import, is_package):
```

Comment:
If not private, docstrings are required. You could make this private though.

Reply:
Done